android/KMEA/app/src/main/assets/keyboard.html

```diff
@@ -76,15 +76,15 @@
       if(h > 0) {
         oskHeight = h;
       }
-      var kmw=window['keyman'];
+      var kmw=com.keyman.singleton;  
```

Comment:
Can we avoid using `com.keyman` in new code?

Reply:
I ended up reverting the entire keyboard.html file